seqr/views/apis/individual_api_tests.py

```diff
@@ -1087,6 +1342,10 @@ class LocalIndividualAPITest(AuthenticationTestCase, IndividualAPITest):
     fixtures = ['users', '1kg_project', 'reference_data']
     HAS_EXTERNAL_PROJECT_ACCESS = False
 
+    def test_import_gregor_metadata(self, *args):
+        # Importing gregor metadata does not work in local environment
```

Comment:
Is it too much to mock the return value of `file_iter` to yield rows of 'mock' tsv files? I'm curious about what the files look like 😭

Reply:
So the way this test file is structured it has a base `IndividualAPITest` object thats not actually a runnable test case, and then it has `LocalIndividualAPITest` and `AnvilIndividualAPITest` that both inherit from that object plus from a different auth base class so that every test in the `IndividualAPITest` is by default run under both local install and anvil auth conditions. However, this test fails under local conditions because you can't import metadata from anvil buckets without anvil, so for the `LocalIndividualAPITest` we override this function so it doesn't run

Reply:
oh I missed that `test_import_gregor_metadata` is in a different `*APITest` class. thanks for explaining that